libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -451,6 +451,33 @@ void ofFbo::clear() {
 #endif
 }
 
+
+//--------------------------------------------------------------
+void ofFbo::clearColorBuffer(const ofFloatColor & color){
+	glClearBufferfv(GL_COLOR, 0, &color.r);
+}
+
+
+//--------------------------------------------------------------
+void ofFbo::clearColorBuffer(size_t buffer_idx, const ofFloatColor & color){
+	glClearBufferfv(GL_COLOR, buffer_idx, &color.r);
+}
+
+//--------------------------------------------------------------
+void ofFbo::clearDepthBuffer(float value){
+	glClearBufferfv(GL_DEPTH, 0, &value);
+}
+
+//--------------------------------------------------------------
+void ofFbo::clearStencilBuffer(float value){
+	glClearBufferfv(GL_STENCIL, 0, &value);
```

Comment:
Stencil should probably be cleared with `int` value, and `glClearBuffervi` used instead, according to the spec: 
see: https://www.opengl.org/sdk/docs/man/html/glClearBuffer.xhtml
"If buffer is GL_STENCIL, drawbuffer must be zero, and value points to a single value to clear the stencil buffer to. Masking is performed in the same fashion as for glClearStencil. Only the *iv forms of these commands should be used to clear stencil buffers; be used to clear stencil buffers"